src/server/services/views/secrets.njk

```diff
@@ -0,0 +1,71 @@
+{% extends "layouts/tabbed-page.njk" %}
+
+{% from "icons/tick-icon/macro.njk" import appTickIcon %}
+
+{% block beforeTabContent %}
+
+  {{ appHeading({
+    title: heading,
+    caption: "Information about the " + heading + " microservice.",
+    entities: headingEntities
+  }) }}
+
+{% endblock %}
+
+{% block tabContent %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-three-quarters">
+
+      {% call appSplitPane() %}
+        <section class="govuk-!-margin-top-2 govuk-!-margin-left-2">
+          <h2 class="govuk-heading-l govuk-!-margin-bottom-2">Secrets</h2>
+          <p class="govuk-!-margin-bottom-2">
+            Perform the following actions for service secrets in any environments.
+          </p>
+
+          <ul class="govuk-list govuk-list--bullet">
+            <li>
+              <div class="app-!-layout-centered">
+                Create
+                {{ appTickIcon({ classes: "app-icon-tiny govuk-!-margin-left-1" }) }}
+              </div>
+            </li>
+            <li>
+              <div class="app-!-layout-centered">
+                Update
+                {{ appTickIcon({ classes: "app-icon-tiny govuk-!-margin-left-1" }) }}
+              </div>
+            </li>
+            <li>
+              <div class="app-!-layout-centered">
+                Check
+                {{ appTickIcon({ classes: "app-icon-tiny govuk-!-margin-left-1" }) }}
+              </div>
+            </li>
+            <li>
+              <div class="app-!-layout-centered">
+                Delete
+                {{ appTickIcon({ classes: "app-icon-tiny govuk-!-margin-left-1" }) }}
+              </div>
```

Comment:
Do you want to comment out these "check" and "delete" copy text from the front secrets page? Until we add those in?